airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -41,6 +41,8 @@
     DestinationType.POSTGRES.value: 63,
     # https://dev.mysql.com/doc/refman/8.0/en/identifier-length.html
     DestinationType.MYSQL.value: 64,
+    # find source
```

Comment:
Is this a TODO?

Reply:
https://oracle-base.com/articles/12c/long-identifiers-12cr2 I'll add this